financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryScreen.kt

```diff
@@ -53,14 +55,15 @@ internal fun ManualEntryScreen() {
     val viewModel: ManualEntryViewModel = mavericksViewModel()
     val parentViewModel = parentViewModel()
     val state: ManualEntryState by viewModel.collectAsState()
+    val form by viewModel.form.collectAsState()
     ManualEntryContent(
-        routing = state.routing,
-        routingError = state.routingError,
-        account = state.account,
-        accountError = state.accountError,
-        accountConfirm = state.accountConfirm,
-        accountConfirmError = state.accountConfirmError,
-        isValidForm = state.isValidForm,
+        routing = viewModel.routing ?: "",
```

Comment:
Should we expose this as non-optional strings instead?

Reply:
https://github.com/stripe/stripe-android/pull/8060/commits/2d8f6c015c317135cdf0ce70b35baace31a7af4e